lib/routes/weibo/timeline.js

```diff
@@ -16,143 +41,70 @@ module.exports = async (ctx) => {
-                    try {
-                        const rep = await got.get(`https://api.weibo.com/2/emotions.json?access_token=${token}&type=${type}`);
-                        rep.data.forEach((item) => {
-                            tmpEmotionList[item.phrase] = item.url;
-                        });
-                        return Promise.resolve(rep);
-                    } catch (err) {
-                        // pass
-                    }
-                })
-            );
-            return tmpEmotionList;
-        });
-
-        const data = await Promise.all(
+        const resultItem = await Promise.all(
             response.data.statuses.map(async (item) => {
-                const realContent = await ctx.cache.tryGet('weibotimelineurl' + item.user.id + item.id, async () => {
-                    const res = [];
-                    const videoList = [];
-                    let picUrlList = [];
-                    picUrlList = picUrlList.concat(item.pic_urls);
-                    try {
-                        const rep = await got.get(`https://m.weibo.cn/${item.user.id}/${item.id}`);
-                        rep.body.replace(/"text": "([\s\S]*?)",/g, (s, value) => {
-                            res.push(value.replace(/\\"/g, "'"));
-                        });
-                        rep.body.replace(/"mp4_720p_mp4": "([\s\S]*?)",/, (s, value) => {
-                            videoList.push(value);
-                        });
-                        if (res.length > 1) {
-                            const sub_rep = await got.get(`https://m.weibo.cn/${item.retweeted_status.user.id}/${item.retweeted_status.id}`);
-                            sub_rep.body.replace(/"text": "([\s\S]*?)",/g, (s, value) => {
-                                res[1] = value.replace(/\\"/g, "'");
-                            });
-                            sub_rep.body.replace(/"mp4_720p_mp4": "([\s\S]*?)",/, (s, value) => {
-                                videoList.push(value);
-                            });
-                            picUrlList = picUrlList.concat(item.retweeted_status.pic_urls);
-                        }
-                    } catch (err) {
-                        res.push(item.text);
-                        if (item.retweeted_status) {
-                            res.push(item.retweeted_status.text);
-                            picUrlList = picUrlList.concat(item.retweeted_status.pic_urls);
-                        }
-                        // 非全文获取的微博内容需要替换表情
-                        res.forEach((content, index) => {
-                            const emotion = [];
-                            content.replace(/(\[.*?\])/g, (s, value) => {
-                                emotion.push(value);
-                            });
+                let data = {};
+                const key = `weibotimelineurl${item.user.id}${item.id}`;
+                const value = await ctx.cache.get(key);
```

Comment:
Please use cache.tryGet() instead of cache.set/cache.get

docs: https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben

Reply:
sry, my fault